src/components/EditActivityPage/EditActivityPage.js

```diff
@@ -69,23 +72,25 @@ const EditActivityPage = ({
   useEffect(asyncSetActivity, [id]);
 
   return (
-    <div>
+    <ActivityFormWrapper>
+      <Heading as="h1">Edit Activity</Heading>
       <Loader loaded={isIdLoaded} data-test="loader">
         <ActivityForm
           activity={editedActivity}
           onSubmit={activitySubmited => asyncEditActivity(activitySubmited)}
           data-test="form"
         />
+        <Button
+          onClick={() => asyncRemoveActivity(activityId)}
```

Comment:
not for this pr, but can we change this function name to normal one ? why not `removeActivity` ?